src/primitives/box.jl

```diff
@@ -33,6 +33,23 @@ measure(b::Box) = prod(b.max - b.min)
 diagonal(b::Box) = norm(b.max - b.min)
 sides(b::Box) = b.max - b.min
 
+"""
+Return all corners of the box.
+"""
+@generated function vertices(b::Box)
+  list = sort(collect(distinct(IterTools.subsets(repeat(1:2, paramdim(b)), Val{paramdim(b)}()))))
```

Comment:
Can you elaborate or maybe split this line into multiple lines? It is a bit complicated to grasp, at least reading it for the first time. Perhaps split into multiple lines explaining what is going on.

Reply:
Isn't it clear? We take the sorted version of distinct subsets of indices 1:2 repeated over the parametric dimension of b :laughing:

Reply:
🤣 now that you read it out loud I can start processing it hahaha. These codes with fancy iterations are always tricky to grasp.